doc/security.md

```diff
@@ -36,6 +36,61 @@ will become world-readable.
 * `/run/systemd/system/netplan-wpa-*.service`
 * `/run/systemd/system/systemd-networkd-wait-online.service.d/10-netplan*.conf`
 
+## Cryptography
+
+Netplan does not directly utilise cryptography, but configures underlying tools
+to do so. Such tools include `wpa_supplicant`, `systemd-networkd`, `NetworkManager`
+or `Open vSwitch` and they can for example be configured to setup WPA2/WPA3
+[encypted WiFi](https://w1.fi/wpa_supplicant/devel/code_structure.html#crypto_func) 
+connections, `802.1x` encrypted Ethernet connections,
```

Comment:
We probably should reword this as something like `802.1x for wired and wireless authentication and authorization,` as 802.1x alone doesn't provide encryption of network traffic.